.gitignore

```diff
@@ -1,3 +1,3 @@
-node_modules
+/node_modules
```

Comment:
seems like we need `/test/foo/node_modules`:
![foo](https://user-images.githubusercontent.com/6443576/27910591-2f104d1a-622c-11e7-8d67-009e16ba5c99.png)